src/core/StelTexture.hpp

```diff
@@ -20,9 +20,7 @@
 #ifndef STELTEXTURE_HPP
 #define STELTEXTURE_HPP
 
-#include "StelTextureTypes.hpp"
```

Comment:
I wonder why this builds for you and in the CI, because I get the following errors:
```
In file included from /home/ruslan/src/stellarium/src/core/StelTexture.cpp:21:
/home/ruslan/src/stellarium/src/core/StelTextureMgr.hpp:47:2: error: ‘StelTextureSP’ does not name a type; did you mean ‘StelTexture’?
   47 |  StelTextureSP createTexture(const QString& filename, const StelTexture::StelTextureParams& params=StelTexture::StelTextureParams());
      |  ^~~~~~~~~~~~~
      |  StelTexture
/home/ruslan/src/stellarium/src/core/StelTextureMgr.hpp:50:2: error: ‘StelTextureSP’ does not name a type; did you mean ‘StelTexture’?
   50 |  StelTextureSP createTexture(const QImage &image, const StelTexture::StelTextureParams& params=StelTexture::StelTextureParams());
      |  ^~~~~~~~~~~~~
      |  StelTexture
/home/ruslan/src/stellarium/src/core/StelTextureMgr.hpp:58:2: error: ‘StelTextureSP’ does not name a type; did you mean ‘StelTexture’?
   58 |  StelTextureSP createTextureThread(const QString& url, const StelTexture::StelTextureParams& params=StelTexture::StelTextureParams(), bool lazyLoading=true);
      |  ^~~~~~~~~~~~~
      |  StelTexture
/home/ruslan/src/stellarium/src/core/StelTextureMgr.hpp:64:2: error: ‘StelTextureSP’ does not name a type; did you mean ‘StelTexture’?
   64 |  StelTextureSP wrapperForGLTexture(GLuint texId);
      |  ^~~~~~~~~~~~~
      |  StelTexture
/home/ruslan/src/stellarium/src/core/StelTextureMgr.hpp:69:2: error: ‘StelTextureSP’ does not name a type; did you mean ‘StelTexture’?
   69 |  StelTextureSP getDitheringTexture(const int samplerToBindTo);
      |  ^~~~~~~~~~~~~
      |  StelTexture
/home/ruslan/src/stellarium/src/core/StelTextureMgr.hpp:87:2: error: ‘StelTextureSP’ does not name a type; did you mean ‘StelTexture’?
   87 |  StelTextureSP lookupCache(const QString& file);
      |  ^~~~~~~~~~~~~
      |  StelTexture
/home/ruslan/src/stellarium/src/core/StelTextureMgr.hpp:93:2: error: ‘StelTextureSP’ does not name a type; did you mean ‘StelTexture’?
   93 |  StelTextureSP ditheringTexture;
      |  ^~~~~~~~~~~~~
      |  StelTexture
```

Reply:
I've fixed it now, but I still wonder why this worked for you.

Reply:
Huh? Interesting. 
What about the PCH setting? I hope this get rebuilt correctly when the relevant headers are touched!

Reply:
Oh, interesting. I have PCH disabled, and indeed the CI seems to have them enabled. So looks like you shouldn't remove headers unless you test build without PCH before pushing.

Reply:
But why does the build system not recreate PCHs when the relevant headers change? Sorry for that. I was just assuming there is a `make` rule that ensures this.

Reply:
The PCHs are only created when `ENABLE_PCH` is true. The `stelMain_pch.hpp` file is only used in this case. Without PCH each source file has to include the required headers one by one.

Reply:
Yes, sure. But I would assume that the PCH should be rebuilt if any of the included headers changes. And BTW, StelTexture(Types).h* is not included, so where did this come from? 
OK, must remember to test without PCH before removing a header.

Reply:
> But I would assume that the PCH should be rebuilt if any of the included headers changes.

Yes, of course they should. But if they are not used, this doesn't affect the build :)

> And BTW, StelTexture(Types).h* is not included, so where did this come from?

`StelCore.hpp` → `StelSkyDrawer.hpp` → `StelTextureTypes.hpp`